test/lib.sh

```diff
@@ -18,10 +18,18 @@ export NEXUS_MK=boilerplate/generated-includes.mk
 _BP_TEST_TEMP_DIRS=
 
 err() {
-  echo "==ERROR== $@" >&2
+  echo -e "\e[31m==ERROR== $@\e[0m" >&2
```

Comment:
I'd suggest defining these ANSI colors to a variable so we know what color this is.

Reply:
Yeah, there's actually a way to use some kind of color name here instead. I'll look into it.